deploy/kubernetes/deploy-kiali-to-kubernetes.sh

```diff
@@ -200,22 +200,28 @@ echo "=== SETTINGS ==="
 
 YAML_DIR=${YAML_DIR:-$(cd "$(dirname "$0")" && pwd -P)}
 
-# Now deploy all the Kiali components to kubernetes
-# If we are missing one or more of the yaml files, download them
-echo "Deploying Kiali to Kubernetes project ${NAMESPACE}"
-for yaml in secret configmap serviceaccount clusterrole clusterrolebinding deployment service ingress crds
-do
-  yaml_file="${yaml}.yaml"
-  yaml_path="${YAML_DIR}/${yaml}.yaml"
+apply_yaml() {
+  local yaml_file="${1}.yaml"
+  local yaml_path="${YAML_DIR}/${yaml_file}"
+  local yaml_url="https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/openshift/${yaml_file}"
```

Comment:
this is the "deploy-kiali-to-kubernetes.sh" script pointing to openshift directory, I guess it's not intended. I'll fix it in my related PR #857 but you might want to fix quicker/separately

Reply:
Fixed in #875